services/QuillLMS/app/controllers/cms/school_admins_controller.rb

```diff
@@ -42,7 +42,13 @@ def set_school
 
   private def handle_school_admin_save(user, school_id, new_user)
     determine_school_admin_worker(user, school_id, new_user)
-    returned_message = new_user ? t('admin.make_admin') : t('admin_created_account.existing_account.admin.new')
+    returned_message = t('admin_created_account.existing_account.admin.new')
```

Comment:
This particular method has become complicated by virtue of the returned_message logic.  I would break out a new method:


```
private def handle_school_admin_save(user, school_id, new_user)
    determine_school_admin_worker(user, school_id, new_user)
    
    if params[:is_make_admin_button]
      flash[:success] = t('admin.make_admin')
      redirect_to cms_school_path(school_id)
    else
      render json: { message: school_admin_save_message(new_user, user) }, status: 200
    end
  end

  private def school_admin_save_message(new_user, user)
    if new_user
      t('admin.make_admin')
    elsif user.is_admin_to_multiple_schools?  # note that this is a new method to write for user.rb
      t('admin_created_account.existing_account.admin.admin_for_other_school')
    else
      t('admin_created_account.existing_account.admin.new')
    end
  end
```

Reply:
See [comment](https://github.com/empirical-org/Empirical-Core/pull/10211#discussion_r1089299382) for note on `user.is_admin_to_multiple_schools?`

Reply:
yeah, that seems cleaner-- thanks for the suggestion!